server-go/storage.go

```diff
@@ -19,6 +19,16 @@ import (
 	opspb "google.golang.org/genproto/googleapis/longrunning"
 )
 
+type ListOptions struct {
```

Comment:
@vtsao Do you have thoughts on this struct? I have no issues with it.

Reply:
I would just inline these fields as args to the method, it makes it easier to read and I don't think we expect this to grow anytime soon.

Reply:
Fixed in 724e2e3